dataservice/api/biospecimen/models.py

```diff
@@ -102,3 +103,7 @@ class Biospecimen(db.Model, Base):
                                     backref=db.backref(
                                         'biospecimen', lazy=True),
                                     doc='genomic files this biospecimen')
+    diagnoses = db.relationship(Diagnosis,
```

Comment:
Do we really want this delete cascade?  If you delete a biospecimen, the corresponding diagnoses might still be attached to the participant and we may still want them stored.